discovery_engine_core/web-api/openapi.yaml

```diff
@@ -0,0 +1,160 @@
+openapi: 3.0.3
+info:
+  title: Personalization API
+  version: 1.0.0-rc1
+  description: |-
+    The endpoints can be contacted at `t<name>.xaynet.dev/v1` or `t<name>.xaynet.dev/v1.0`.
+    The `v1` path will always point to the latest `v1.x`.
```

Comment:
`t` for `tenant` but could also be different. I think it could be useful for us to have some kind of template in case we want to automatize some part of the domain.

For the second part or we do it or we don't. No one want to point to and endpoint that has no guaranteed of existing.

Reply:
I left only `v1`

Reply:
Actually I decided to remove this part completely. Is not that relevant at the moment, we still need to decide which domain to use and I don't want to commit to information what we might change later and are not directly relevant for the api.